docs/_includes/ds6/icon.html

```diff
@@ -20,8 +20,8 @@ <h3>Inline SVG</h3>
         {% endhighlight %}
     </div>
 
-    <h3>Background SVG</h3>
-    <p>Background SVGs have a fixed colour that cannot be changed via CSS.</p>
+    <h3>@ebay/skin/icon/background</h3>
+    <p>Background icons have a fixed colour that cannot be changed via CSS.</p>
```

Comment:
Should we have some warning about the size penalty when using background icons?

Reply:
Good idea. Will add.